src/graphics/chunks.js

```diff
@@ -36,7 +36,7 @@ pc.extend(pc, (function () {
     };
 
 
-    shaderChunks.createShader = function(device, vsName, psName) {
+    shaderChunks.createShader = function(device, vsName, psName, useTransformFeedback) {
```

Comment:
Not sure this API suggests multiple buffers, this is static method on TransformFeedback, no more.
```javascript
var tf = new pc.TransformFeedback(buffer);

var shaderA = pc.TransformFeedback.createShader(device, codeA, 'tfShaderA');
var shaderB = pc.TransformFeedback.createShader(device, codeB, 'tfShaderB');

// in somewhere else
tf.process(shaderA);
tf.process(shaderB);
```